### PR TITLE
feat: add loop: false for list type prompts

### PR DESCRIPTION
**Pull request by adiktofsugar**
Fixes: #924

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -210,9 +210,11 @@ Separator instances have a property `type` equal to `separator`. This should all
 
 #### List - `{type: 'list'}`
 
-Take `type`, `name`, `message`, `choices`[, `default`, `filter`] properties. (Note that
+Take `type`, `name`, `message`, `choices`[, `default`, `filter`, `loop`] properties. (Note that
 default must be the choice `index` in the array or a choice `value`)
 
+`loop:false` will disable the default looping behavior.
```
Comment (LitoMore) [inline README.md:216]: You should add put this documentation to [Question](https://github.com/SBoudrias/Inquirer.js#question) section.

File: README.md [hunk 2]
```diff
@@ -222,6 +224,8 @@ default must be the choice `index` in the array or a choice `value`)
 Take `type`, `name`, `message`, `choices`[, `default`, `filter`] properties. (Note that
 default must be the choice `index` in the array)
 
+`loop:false` will disable the default looping behavior.
```
Comment (LitoMore) [inline README.md:227]: Ditto.

File: README.md (hunk 1 above)
Comment (adiktofsugar) [reply README.md]: moved

**Comment by SBoudrias** [comment]
Thanks @adiktofsugar

**Comment by sqren** [comment]
Hey @adiktofsugar 

Thanks for adding this. Just noticed one odd thing: for lists with more items than the `pageSize`, even with `loop:false` the list is still displayed as a loop (the last item "50" is above the first item "1"). Would be great if the loop effect could be disabled entirely:

backport bug2

**Pull request merged**